modules/sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service.adoc

```diff
@@ -0,0 +1,120 @@
+= Connecting the Spring PetClinic sample application to the PostgreSQL database service
+[id="sbo-connecting-spring-petclinic-sample-app-to-postgresql-database-service_{context}"]
+
+For connecting the sample application to the database service, you have to create a `ServiceBinding` custom resource (CR) that triggers the {servicebinding-title} to project the binding data into the application.
```

Comment:
```suggestion
To connect the sample application to the database service, create a `ServiceBinding` custom resource (CR) that triggers the {servicebinding-title} to project the binding data into the application.
```
Grammatical usage: "To connect" rather than "For connecting"

Also, per IBM usage, consider either changing the main clause to the imperative our using "you must." See the "have to" entry in the _IBM Style Guide_.